File: odd_even.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

console.log("Enter numbers (type 'quit' to stop):");

rl.on("line", (input) => {
  if (input.toLowerCase() === "quit") {
    console.log("Stream ended.");
    rl.close();
  } else {
    let num = Number(input);
    if (!isNaN(num)) {
      console.log(num + " → " + (num % 2 === 0 ? "Even" : "Odd"));
    } else {
      console.log("Enter a valid number or 'quit'.");
    }
  }
});

File: misc/odd_even.cpp
#include <iostream>
using namespace std;

string oddOrEven(int num) {
    return (num % 2 == 0) ? "Even" : "Odd";
}

int main() {
    cout << oddOrEven(10) << endl; // Output: Even
    cout << oddOrEven(7) << endl;  // Output: Odd
    return 0;
}
